Diagnostics proofreading (#1132)
greenelab/covid19-review: content/10.diagnostics.md

### Abstract

The COVID-19 pandemic has presented a number of challenges that have spurred biotechnological research aiming to address specific problems.

> Diagnostics is one area where biotechnology has been particularly important.<!--Alt: One important area of biotechnology research is diagnostics.-->

Reviewer: This line was rewritten here and in #1120. I will default to the version that is not in the comments unless there is a preference for the other!

> Diagnostics is one area where biotechnology has been particularly important.<!--Alt: One important area of biotechnology research is diagnostics.-->

Reviewer: This line was rewritten here and in #1120. I will default to the version that is not in the comments unless there is a preference for the other!

Reviewer 2: I think this is fine, thank you.

This design allows for a digital read-out where each partition is either positive or negative for the nucleic acid sequence being tested for, allowing for absolute target quantification through Poisson statistics.
While dPCR equipment is not yet as common as that for qPCR, dPCR for DNA targets generally achieves higher sensitivity than other PCR technologies while maintaining high specificity, though sensitivity is slightly lower for RNA targets [@doi:10.3390/s18041271].
High sensitivity is particularly relevant for SARS-CoV-2 detection, since low viral load in clinical samples can lead to false negatives.
Suo et al. [@doi:10.1080/22221751.2020.1772678] performed a double-blind evaluation of ddPCR for SARS-CoV-2 detection on 57 samples, 43 samples from suspected positive patients and 14 from supposed convalescents, that had all tested negative for SARS-CoV-2 using qPCR.

> Despite the initial negative results, 33 out of 35 (94.3%) patients were later clinically confirmed positive. <!-- Where did the 35 come from? -->

Reviewer 3: `33 out of 35` is wrong in the current context, the numbers come from the first version of the preprint (https://www.medrxiv.org/content/10.1101/2020.02.29.20029439v1). I think the numbers in the entire section need to be double-checked.

> Despite the initial negative results, 33 out of 35 (94.3%) patients were later clinically confirmed positive. <!-- Where did the 35 come from? -->

Author: Changed in 6fc72cabd551be3ea29cfc215e15b68841a214da

Sequencing efforts are important for identifying variants containing mutations that might affect the reliability of molecular diagnostic tests, as well as mitigation measures such as therapeutics and prophylactics [@doi:10.1038/s41576-021-00360-w; @doi:10.2807/1560-7917.ES.2020.25.39.2001650].
Therefore, sequencing is an important component of diagnostics: while it is not necessary for diagnosing an individual case, it is critical to monitoring trends in the variants affecting a population and to staying aware of emerging variants that may pose additional challenges.

##### Pooled and Automated PCR Testing

Due to limited supplies and the need for more tests, several labs have found ways to pool or otherwise strategically design tests to increase throughput.
The first such result came from Yelin et al. [@doi:10.1093/cid/ciaa531], who reported that they could pool up to 32 samples in a single qPCR run.
This was followed by larger-scale pooling with slightly different methods [@doi:10.1101/2020.04.02.022186].
Although these approaches are also PCR based, they allow for more rapid scaling and higher efficiency for testing than the initial PCR-based methods developed.

> Conceptually, pooling could also be employed in analysis with RT-qPCR [@doi:10.1109/ICASSP39728.2021.9413685]. <!-- This looks like algorithmic work that doesn't have a proof of concept. Is my rephrasing okay? -->

Reviewer 3: Can't tell whether pooling is currently done in practice, but this has at least been extensively studied. See e.g. https://www.thelancet.com/journals/eclinm/article/PIIS2589-5370(21)00362-X/fulltext (plenty more pop up with `rt-qpcr pooling covid` in Google) ; not sure the currently cited work is the best to use here, since, as you say, it appears to be a specific algorithm for pooling.

This method was benchmarked against RT-PCR using 177 human nasopharyngeal RNA samples, of which 126 were COVID positive.
The authors break down the sensitivity of their test according to the C~t~ value from RT-PCR of the same samples; RT-LAMP performs at 100% sensitivity for samples with a C~t~ from RT-PCR of 32 or less.
The performance is worse when considering all RT-PCR positive samples (including those with C~t~ values between 32-40).
However, there is some evidence suggesting that samples obtained from individuals that achieve C~t~ values >30 measured using RT-PCR tend to be less infective that those that record a C~t~ value <30 [@doi:10.1016/j.jiph.2021.08.013; @doi:10.1017/ice.2021.132; @doi:10.1093/cid/ciaa619], so RT-LAMP is still a useful diagnostic tool.
Various combinations of reagents are available, but one example is the WarmStart Colorimetric LAMP 2X Master Mix with a set of six primers developed previously by Zhang et al. [@doi:10.1101/2020.02.26.20028373].

> To determine assay sensitivity, serial tenfold dilutions of _in vitro_ transcribed N-gene RNA standard were tested using quantities from 10^5^ copies down to 10 copies.

Reviewer 3: This is the only instance where we don't italicize N. I think it should be it since it refers to the gene sequence here, not the protein.

Author: Fixed in c1298eea0dcc728014917c3ca9428558eba98072

The assay readout is the color of the dye changing from pink to yellow due to binding to the DNA product over 30 minutes.
The RT-LAMP assay was then applied to clinical nasopharyngeal samples.
For viral loads above 100 copies of genomic RNA, the RT-LAMP assay had a sensitivity of 100% and a specificity of 96.1% from purified RNA.

> The direct assay of saliva by RT-LAMP had a sensitivity of 85% (CI 70-93%).

Reviewer: ```suggestion
The sensitivity of the direct assay of saliva by RT-LAMP was 85%.
```

Sensitivity and specificity metrics were obtained by comparison with results from RT-PCR.
RT-LAMP pilot studies for detection of SARS-CoV-2 were reviewed in a meta-analysis [@PMID:34086539].

> In the meta-analysis of all 2,112 samples, the cumulative sensitivity of RT-LAMP was 95.5% (CI 97.5% = 90.8-97.9%) and the cumulative specificity was 99.5% (CI 97.5% = 97.7-99.9%). <!-- This is different confidence interval notation than we've used elsewhere -->

Reviewer: Grep on "CI" suggests we usually don't mention them (I only got 4 lines) so I am going to make this less precise.

```suggestion
In the meta-analysis of all 2,112 samples, the cumulative sensitivity of RT-LAMP was calculated at 95.5%, and the cumulative specificity was 99.5%.
```

LFTs provide distinct value relative to PCR tests.
They can return results within 30 minutes and can be performed without specialized equipment and at low cost.
They also do not require training to operate and are cheap to produce.
Thus, they can be distributed widely to affected populations making them an important public health measure to curb pandemic spread.
LFTs rely on the detection of viral protein with an antibody.
Often this is done with an antibody sandwich format, where one antibody conjugated to a dye binds at one site on the antigen, and an immobilized antibody on the strip binds at another site [@doi:10.1007/s00216-010-3661-4].
This design allows the dye to accumulate to form a characteristic positive test line on the strip [@doi:10.1007/s00216-010-3661-4].
Outside of COVID-19 diagnostics, the applications of LFTs are broad; they are routinely used for home pregnancy tests, disease detection, and even drugs of abuse detection in urine [@DOI:10.1007/s00216-008-2287-2].

A recent review surveyed the performance of LFTs for detection of current SARS-CoV-2 infection [@PMID:34407759].
This review covered 24 studies that included more than 26,000 total LFTs.

> They found significant heterogeneity in test sensitivities ranging from 37.7% (95% CI 30.6-45.5%) to 99.2% (95% CI 95.5-99.9%).

Reviewer: ```suggestion
They reported significant heterogeneity in test sensitivities, with estimates ranging from 37.7% to 99.2%.
```

> The specificities of these tests were more homogeneous, spanning 92.4% (95% CI 87.5-95.5%) to 100.0% (95% CI 99.7-100.0%). <!-- Other CIs don't have the 95% -->

Reviewer: ```suggestion
The estimated specificities of these tests were more homogeneous, spanning 92.4% to 100.0%.
```

For COVID-19 diagnostics, ELISAs have been designed to detect the antigenic Spike protein [@doi:10.12688/wellcomeopenres.15927.1].

One of these assays uses two monoclonal antibodies specific to the nucleocapsid of SARS-CoV-2 to evaluate the relationship between the effect of (estimated) viral load on the ability of the assay to detect the SARS-CoV-2 antigen [@doi:10.1080/14787210.2021.1976144].
This study analyzed 339 naso-oropharyngeal samples that were also analyzed with RT-qPCR as a gold standard.
RT-qPCR identified 147 samples as positive and 192 as negative.
The authors estimated the overall sensitivity and specificity to be 61.9% and 99.0%, respectively.
Sensitivity increased with higher C~t~.
This study also assessed the performance of the ELISA test under different conditions in order to evaluate how robust it would be to the challenges of testing in real-world settings globally.
Higher sensitivity was achieved for samples that were stored under ideal conditions (immediate placement in -80° C).
Therefore, while immediate access to laboratory equipment is an advantage, it is not strictly necessary for ELISA to detect the antigen.

#### Limitations of Molecular Tests

> Tests that identify SARS-CoV-2 using nucleic-acid-based technologies will identify only individuals with current infections and are not appropriate for identifying individuals who have recovered from a previous infection. <!-- Isn't this also true for antigen-based tests discussed above? -->

Reviewer: ```suggestion
Tests that identify SARS-CoV-2 using molecular technologies will identify only individuals with current infections and are not appropriate for identifying individuals who have recovered from a previous infection.
```

> Among molecular tests, different technologies have different sensitivities and specificities. <!-- Specificity and sensitivity are so central to the paper that it seems like the terms should be introduced early in the manuscript -->

Reviewer: Great suggestion! #1134

The application of ELISA to serological testing is complementary to its use in molecular diagnostics (see above).
Instead of using an enzyme-labeled antibody as a probe that binds to the target antigen, the probe is an antigen and the target is an antibody.
In March 2020, the Krammer lab proposed an ELISA test that detects IgG and IgM that react against the receptor-binding domain (RBD) of the spike proteins (S) of the virus [@doi:10.1038/s41591-020-0913-5].

> A subsequent ELISA test developed to detect SARS-CoV-2 IgG based on the RBD reported a selectivity of over 99% and a sensitivity of up to 88.24%, which was observed in samples collected 21 to 27 days after the onset of infection (approximated with symptom onset or positive PCR test) [@doi:10.3389/fmicb.2020.618097]. <!-- I suggest not using selectivity here or elsewhere, we should standardize terminology if that is meant as a synonym of specificity -->

Reviewer 3: Agreed, that's why I changed the few instances to specificity in the other PR. This seems to inadvertently revert it?

Author: Fixed in c1298eea0dcc728014917c3ca9428558eba98072

Using computed tomography of the chest in addition to RT-qPCR testing was found to provide a higher sensitivity than either measure alone [@doi:10.1101/2020.02.25.20027755].
X-ray diagnostics have been reported to have high sensitivity but low specificity in some studies [@doi:10.1148/radiol.2020200642].
Other studies have shown that specificity varies between radiologists [@doi:10.1148/radiol.2020200823], though the sensitivity reported here was lower than that published in the previous paper.

> However, preliminary machine-learning results have shown far higher sensitivity and specificity from analyzing chest X-rays than was possible with clinical examination [@doi:10.1007/s13246-020-00865-4]. <!-- Do we take these results at face value? Studies like https://doi.org/10.1038/s42256-021-00307-0 have questioned whether ML models like this have any clinical value. I'm not sure whether the surveyed this particular paper we cite here, but some additional caveats or skepticism may be needed (in the next sentence too). -->

Reviewer 3: Strongly agreed; if we decide to mention ML models here they should be discussed a bit more thoroughly.

Reviewer: @mprobson any interest?

Reviewer 4: Taking a look, I'm not sure I'm the right person to address the potential deficiencies in ML models for medicine but I agree some extra context and caveats are warranted.

Reviewer: ```suggestion
While preliminary machine-learning results suggested that chest X-rays might provide high sensitivity and selectivity and potentially facilitate the detection of asymptomatic and presymptomatic infections (e.g., [@doi:10.1007/s13246-020-00865-4]), further investigation suggested that such approaches are prone to bias and are unlikely to be clinically useful [@doi:10.1038/s42256-021-00307-0].

Using computed tomography of the chest in addition to RT-qPCR testing was found to provide a higher sensitivity than either measure alone [@doi:10.1101/2020.02.25.20027755].
X-ray diagnostics have been reported to have high sensitivity but low specificity in some studies [@doi:10.1148/radiol.2020200642].
Other studies have shown that specificity varies between radiologists [@doi:10.1148/radiol.2020200823], though the sensitivity reported here was lower than that published in the previous paper.
However, preliminary machine-learning results have shown far higher sensitivity and specificity from analyzing chest X-rays than was possible with clinical examination [@doi:10.1007/s13246-020-00865-4]. <!-- Do we take these results at face value? Studies like https://doi.org/10.1038/s42256-021-00307-0 have questioned whether ML models like this have any clinical value. I'm not sure whether the surveyed this particular paper we cite here, but some additional caveats or skepticism may be needed (in the next sentence too). -->

> X-ray tests with machine learning can potentially detect asymptomatic or presymptomatic infections that show lung manifestations.
> However, this approach would still not recognize entirely asymptomatic cases.

Reviewer: ```suggestion

```

Using computed tomography of the chest in addition to RT-qPCR testing was found to provide a higher sensitivity than either measure alone [@doi:10.1101/2020.02.25.20027755].
X-ray diagnostics have been reported to have high sensitivity but low specificity in some studies [@doi:10.1148/radiol.2020200642].
Other studies have shown that specificity varies between radiologists [@doi:10.1148/radiol.2020200823], though the sensitivity reported here was lower than that published in the previous paper.

> While preliminary machine-learning results suggested that chest X-rays might provide high sensitivity and selectivity and potentially facilitate the detection of asymptomatic and presymptomatic infections (e.g., [@doi:10.1007/s13246-020-00865-4]), further investigation suggested that such approaches are prone to bias and are unlikely to be clinically useful [@doi:10.1038/s42256-021-00307-0]. 

Reviewer 3: ```suggestion
While preliminary machine-learning results suggested that chest X-rays might provide high sensitivity and specificity and potentially facilitate the detection of asymptomatic and presymptomatic infections (e.g., [@doi:10.1007/s13246-020-00865-4]), further investigation suggested that such approaches are prone to bias and are unlikely to be clinically useful [@doi:10.1038/s42256-021-00307-0]. 
```
